tests/Unit/Evolution/Systems/Cce/CceComputationTestHelpers.hpp

```diff
@@ -0,0 +1,59 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include "DataStructures/ComplexDataVector.hpp"
+#include "DataStructures/ComplexModalVector.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tags.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Evolution/Systems/Cce/Tags.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "NumericalAlgorithms/Spectral/SwshCoefficients.hpp"
+#include "NumericalAlgorithms/Spectral/SwshFiltering.hpp"
+#include "NumericalAlgorithms/Spectral/SwshTags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/VectorAlgebra.hpp"
+#include "tests/Utilities/MakeWithRandomValues.hpp"
+
+namespace Cce {
+namespace TestHelpers {
+
+// For representing a primitive series of powers in inverse r for diagnostic
+// computations
+template <typename T>
+struct RadialPolyCoefficientsFor : db::SimpleTag {
+  // for the test, we just store a quadratic in radius
+  using type = Scalar<ComplexModalVector>;
+  using tag = T;
+  static std::string name() {
+    return "RadialPolyCoefficientsFor(" + T::name() + ")";
+  }
+};
+
+// For representing the angular function in a separable quantity.
+template <typename T>
+struct AngularCollocationsFor : db::SimpleTag {
+  using type = typename T::type;
+  using tag = T;
+  static std::string name() {
+    return "AngularCollocationsFor(" + T::name() + ")";
+  }
+};
+
+// shortcut method for evaluating a frequently used radial quantity in CCE
+void compute_one_minus_y(
```

Comment:
Remove the `compute` prefix.

Also, would it make sense to add this to the `src`, or do you need it only in the tests?

Reply:
only needed for tests, true computations will use a struct in a future PR.